Angular/src/app/app.component.ts

```diff
@@ -1,38 +1,41 @@
 import { Component } from '@angular/core';
+import { DataRowTemplateData } from 'devextreme/ui/data_grid';
 import {
   GridDataModel,
   DynamicType,
   MyCustomType,
   Service,
 } from './app.service';
-import { DataRowTemplateData } from 'devextreme/ui/data_grid';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css'],
 })
 export class AppComponent {
+  service: Service;
+
   sampleData: GridDataModel[];
+
   typeList: string[];
 
-  constructor(private service: Service) {
+  constructor(service: Service) {
```

Comment:
Why are we changing this?
Do we have any eslint rule that forbids the old code version?

In my point of view, it's better to use:
```typescript
  constructor(private service: Service) { }
```

instead of the:
```typescript
private service: Service;

constructor(service: Service) {
  this.service = service
}
```

Reply:
Yes, I have to change this due to the **@typescript-eslint/prefer-readonly** rule as I remember

Reply:
See https://github.com/markallenramirez/devextreme-datagrid-dynamic-type-column/actions/runs/5185457659/jobs/9345390723#step:6:46

Reply:
Yup, got it
Let's discuss this rule separately and leave it as is for now